src/assetExercise.js

```diff
@@ -6,6 +6,13 @@ const logger = require('./log')(__dirname)
 
 const router = express.Router()
 
+const getSafeName = (name) => (/^\w+\.?\w+$/.test(name) ? name : null)
```

Comment:
Could you add a comment explaining the regex please? This way if somebody goes comes back to this code they don't need to try and read the regex to understand it.

Also, since you just return the name if the test passes, it'd better to return a boolean instead. Maybe `isSafePath` would be a better name too since it will be used to define the file path.

```js
const isSafePath = (name) => /^\w+\.?\w+$/.test(name)
```

And then on the router method you do this:

```js
const { name } = req.body
if (!isSafePath(name)) return sendUnsafeNameErrorResponse(res, name)
```